packages/roosterjs-editor-dom/lib/table/VTable.ts

```diff
@@ -63,6 +78,17 @@ export default class VTable {
                                 width: hasTd ? rect.width : undefined,
                                 height: hasTd ? rect.height : undefined,
                             };
+
+                            if (
+                                setSelectedRange &&
+                                td.classList.contains(TableMetadata.TABLE_CELL_SELECTED)

```

Comment:
Does this work in IE11?

For now roosterjs still supports IE11. We can stop the supporting in next major release

Reply:
Yes, IE11 only have partial support when using classlist on SVG elements
https://developer.mozilla.org/en-US/docs/Web/API/Element/classList
![image](https://user-images.githubusercontent.com/8291124/150605112-8dbf5734-2385-44e3-94c7-700be44d2521.png)